projects/pastanaga-angular/src/lib/toast/toast.service.ts

```diff
@@ -43,7 +45,8 @@ export class ToastService {
         this.open(message, 'error', config);
     }
 
-    private open(message: string, type: ToastType, config?: ToastConfig) {
+    open(message: string, type: ToastType, config?: ToastConfig) {
```

Comment:
why do you set this method as public?

Reply:
because I want to be able to use it directly